### PR TITLE
[2.19.x] DDF-5829: Added plugin support to map.view; fixed OpenLayers minZoom

### PR DESCRIPTION
**Pull request by leo-sakh**
#### What does this PR do?
Adds ability to extend map (Cesium and OpenLayers) config functionality by downstream projects via 'plugin'

#### Who is reviewing it? 
@nsuvarna
@bennuttle 
@aj-brooks 
@AzGoalie 
@rymach 
@lavoywj 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@AzGoalie
@bdeining

#### How should this be tested?
Interact with both 2D and 3D maps in the UI to verify no regressions have been introduced.

#### What are the relevant tickets?
Fixes: #5829

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

**Comment by leo-sakh** [comment]
build now

**Comment by leo-sakh** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js [hunk 1]
```diff
@@ -188,11 +189,12 @@ module.exports = function CesiumMap(
...
   notificationEl,
   componentElement,
-  mapModel
+  mapModel,
+  cameraOptions
 ) {
   let overlays = {}
   let shapes = []
-  const map = createMap(insertionElement)
+  const map = createMap(insertionElement, cameraOptions && cameraOptions.cesium)
```
Comment (millerw8) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js]: This is sending cameraOptions.cesium unless cameraOptions is null, in which case it sends null?

Then when used in createMap, it is cameraOptions?  So cameraOptions.cesium is the type expected by createMap?

**Comment by bdeining** [comment]
verified no regressions

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js (hunk 1 above)
Comment (leo-sakh) [reply ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js]: Yes to both questions.

**Comment by leo-sakh** [comment]
build now

**Pull request merged**